Sources/Suggestions/Score.swift

```diff
@@ -39,7 +40,9 @@ extension Score {
             score += 300
             // Prioritize root URLs most
             if url.isRoot { score += 2000 }
-        } else if lowercasedTitle.starts(with: query) {
+        } else if lowercasedTitle
+                .trimmingCharacters(in: .alphanumerics.inverted)
```

Comment:
This fixes a bug where if a title was like `"Cats and Dogs"` Cats would not get matched at all because of the quote.

Reply:
Yeah that example won't work now because:
* It won't match the first character any more
* The second character wouldn't match anyway

But honestly, I think that's gonna be rare?  I assume people are more likely to search for `HBO` than `"` or `"HBO`

I can make it an OR condition though if you feel strongly?

Reply:
The new solution looks better to me 👍 Thank you, Brindy!